earthpy/tests/test_spatial.py

```diff
@@ -1,19 +1,12 @@
+""" Tests for the spatial module. """
 
-"""Tests for the spatial module"""
-
-from earthpy import spatial as es
 import numpy as np
-import pytest
-
-
 import pandas as pd
-import numpy as np
 import pytest
-from shapely.geometry import Polygon, Point, LineString
-import shapely
+from shapely.geometry import Polygon, Point
 import geopandas as gpd
-import earthpy.spatial as es
-import earthpy.clip as cl
+from earthpy import spatial as es
```

Comment:
i don't normally import this way. why did you adjust this one instance of the import? 

alternatively if we want to be more explicit... could say:

`from earthpy.spatial import extent_to_json, bytescale`

Reply:
Ooops. Changed to `import earthpy.spatial as es`. I like to use `es.some_fun()` over importing particular functions to be super clear in the code where functions are coming from whenever they are called.

Reply:
yaa... i think best python practice is to keep the namespace clean by just importing the functions you want. have you gotten to that part in our OOP book yet? The explicit part comes via the imports! for the spatial module i normally do import  `import earthpy.spatial as es` because often times i use so many functions from it. But like for the show() function in rasterio i just import that fun at the top OR the glob library i normally just import glob. looking at rasterio they are pretty explicit with just importing functions. potential danger zone is two packages, same fun name. i like what you have above but might suggest we change it in the future (or not!) after chatting with a few more folks about namespace best practices :)